config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Folder.java

```diff
@@ -31,11 +32,4 @@ public void setPath(String path) {
         this.path = path;
     }
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add the "@Override" annotation above this method signature ([squid:S1161](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1161))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=122&issues=AWblzez3SgN4uCclhZKg&open=AWblzez3SgN4uCclhZKg)

[ISSUE_KEY]: AWblzez3SgN4uCclhZKg